caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -482,7 +496,7 @@ public function additional_legend_items($items)
         }
         if (EE_Registry::instance()->CAP->current_user_can('ee_read_global_messages', 'view_filtered_messages')) {
             $related_for_icon = EEH_MSG_Template::get_message_action_icon('see_notifications_for');
-            if (isset($related_for_icon['css_class']) && isset($related_for_icon['label'])) {
+            if (is_array($related_for_icon) && isset($related_for_icon['css_class'], $related_for_icon['label'])) {
```

Comment:
`$related_for_icon` can sometimes be a `string` so `'css_class'` would be an illegal offset (can only use numeric offsets when treating strings as arrays)